docs/legal_hold.md

```diff
@@ -0,0 +1,117 @@
+Legal Holds Policy
+==================
+
+Legal Hold Policy information describes the basic characteristics of the Policy,
```

Comment:
This description isn't very informative; what does a Legal Hold Policy do?  What are they for?